docs/reference/configuration/queue.rst

```diff
@@ -183,10 +183,12 @@ The following is a list of available LSF configuration options:
   LSF uses resource requirements to select hosts for remote execution and
   job execution. Resource requirement strings can be simple (applying to the
   entire job) or compound (applying to the specified number of slots).
+  The value passed does not use units and depends on the cluster's configuration,
+  so follow up with cluster administrator to find out what the set unit is.
   `Docs. <https://www.ibm.com/support/knowledgecenter/SSWRJV_10.1.0/lsf_admin/res_req_strings_about.html>`__
   Passed as the ``-R`` option to ``bsub``. For example::
 
-    QUEUE_OPTION LSF LSF_RESOURCE rusage[mem=512MB:swp=1GB]
+    QUEUE_OPTION LSF LSF_RESOURCE rusage[mem=512000:swp=1000000]
```

Comment:
Lets change this to something relevant at Equinor:
`   
   QUEUE_OPTION LSF LSF_RESOURCE rusage[mem=15000]
`
and state this will be (approx) 15 gigabytes when the default unit is megabytes